setup/security/authorization/pipeline-permissions/index.md

```diff
@@ -56,4 +56,36 @@ the previously selected service account in order to maintain backwards
 compatibility. Newly added triggers will use the permissions specified. If you
 wish to use pipeline permissions for the older triggers and already have
 permissions specified, edit your pipeline JSON and remove the `RunAsUser` field
-from your trigger.
+from your trigger, or enable the automatic migration (see next section).
+
+### Automatic migration
+
+Front50 can automatically migrate all pipelines from using [Fiat Service
+Accounts](../service-accounts/) to use Pipeline Permissions and managed service
+accounts. The migrator is disabled by default, and can be enabled by setting
+the following flag:
+
+* Add the following to `front50-local.yml` (`~/.hal/default/profiles/front50-local.yml`
+if you're using Halyard):
```

Comment:
Suggested edit:

"..., and can be enabled by adding the following flag to `front50-local.yml`:

```yaml
migrations:
  migrateToManagedServiceAccounts: true
```
If you're using Halyard, the file is `~/.hal/default/profiles/front50-local.yml`."

Reply:
Fixed